website/assets/js/pages/handbook/basic-handbook.page.js

```diff
@@ -8,6 +8,7 @@ parasails.registerPage('basic-handbook', {
     breadcrumbs: [],
     subtopics: [],
     handbookIndexLinks: [],
+    pagesInSameSection: [],
```

Comment:
This doesn't seem to be in use?  If not in use, it should be omitted.  If it is in use, I'm confused as to how